examples/recipes/README.md

```diff
@@ -0,0 +1,31 @@
+# Recipes
+
+These recipes will show you the best practices of using Cube.js.
+
+### Analytics
+
+- [Daily, Weekly, Monthly Active Users](https://cube.dev/docs/recipes/active-users)
+- [Event Analytics](https://cube.dev/docs/recipes/event-analytics)
+- [Funnel Analysis](https://cube.dev/docs/recipes/funnels)
+- [Retention Analysis](https://cube.dev/docs/recipes/cohort-retention)
+
+### Access control
+
+- [Role-Based Access](https://cube.dev/docs/recipes/role-based-access)
+
+### Data sources
+
+- [Enable SSL Connections to the Database](https://cube.dev/docs/recipes/enable-ssl-connections-to-database)
+
+### Data schema
+
+- [Dynamically Union Tables](https://cube.dev/docs/recipes/dynamically-union-tables)
+- [Data Blending](https://cube.dev/docs/recipes/data-blending)
+- [Snapshots](/recipes/snapshots)
```

Comment:
User will see a 404 page. The only way I found was to specify the full path

Reply:
Good catch!